app/controllers/application.js

```diff
@@ -62,6 +65,12 @@ export default Controller.extend(
 			});
 
 			this._super();
+
+			track({
+				action: trackActions.open,
+				category: 'connection-type',
+				label: `mobile-${this.get('connectionType')}`
```

Comment:
`mobile-` prefix is not necessary. By default we add `mercury-`  prefix to category in mobile-wiki